batcher/aligned-sdk/src/core/types.rs

```diff
@@ -324,7 +325,7 @@ pub enum ResponseMessage {
 }
 
 #[derive(Debug, Clone)]
-pub enum Chain {
+pub enum Network {
```

Comment:
very small nit, but we can derive the `Copy` trait here.
Doing this, the references in the function arguments like `submit_multiple_and_wait_verification` and the others are not needed.
Since it is just an enum, it is very cheap to copy the value (actually cheaper than the reference itself) and the code will look a little cleaner.

Reply:
refactored 👍